web/html/ParamsApp/php/DbSchemaDefs.php

```diff
@@ -200,18 +200,14 @@ function createRequiredDbEntities($dbName, $user, $pw, $dbHost, $quiet = false)
     /* Define any missing req table */
     foreach($reqTables as $tableName => $tableDef) {
 
-        $query = sprintf(TABLE_EXISTS_QUERY, $dbName, $tableName);
-        /* echo "\nquery:\n" . $query; */
-        $result = $conn->query($query);
+        $query = sprintf(TABLE_EXISTS_QUERY, $dbName, $tableName);		// create a query to check if the db and table exist
+        $result = $conn->query($query); 								// perform query
+		if (! $result) {
+			return false;
+		}
         $rs = $result->fetch_array(MYSQLI_ASSOC);
-        /* printf("\nrs['tableExists']: %s", $rs['tableExists']); */
 
-        if ($rs['tableExists'] == "0") {
-            $return .= $tableName . " (created table)\n";
-            /* echo "\ntableDef:\n" . $tableDef; */
-            $result = $conn->query($tableDef);
```

Comment:
Not sure why this is being removed.  If the table doesn't exist, it won't be created.

Reply:
fixed in last commit